GTViewController/Classes/Custom Classes/GTFollowupModalView.m

```diff
@@ -115,7 +97,27 @@ - (instancetype)initFromElement:(TBXMLElement *)element withStyle:(GTPageStyle*)
                 currentY += buttonPairView.frame.size.height + 10;
                 
             }else if ([modalComponentElementName isEqual:kName_Thank_You]) {
-                self.thankYouPage = nil;
+                
+                self.thankYouView = [[GTFollowupThankYouView alloc]initFromElement:modalComponentElement
+                                                                         withFrame:self.frame
+                                                                     withPageStyle:style];
+                
+                NSArray *listeners = [[TBXML valueOfAttributeNamed:kAttr_listeners forElement:modalComponentElement] componentsSeparatedByString:@","];
+                
+                for (id listener in listeners) {
+                    NSString *listenerName = listener;
```

Comment:
Done.  I made the changes both when listeners are received as well as when events are sent.